addons/talo/samples/chat/scripts/chat.gd

```diff
@@ -24,7 +24,8 @@ func _on_identified(player: TaloPlayer) -> void:
 	_subscriptions = await Talo.channels.get_subscribed_channels()
 
 	var res = await Talo.channels.get_channels(0)
-	var channels = res[0]
+	assert(is_instance_valid(res))
+	var channels = res.channels
```

Comment:
is there some GDScript limitation here where async code doesn't pass through the correct type? hovering over `res` I don't see that its a `ChannelPage`

Reply:
GDScript can infer the return value type of async method with `await`. The type of `res` just not be inferred here.
Use ":=" instead of "=" can solve it.

But this is a type safe issue, it can be handled with other untyped code in an independent pr.